javascript/app-framework/src/components/app/DataShaperApp/DataShaperApp.tsx

```diff
@@ -112,7 +118,7 @@ const AppInner: React.FC<DataShaperAppProps> = memo(function AppInner({
 								<Route
 									key={r.href}
 									path={(r.children?.length ?? 0) > 0 ? r.href : `${r.href}/*`}
-									element={<MatchedRoute key={r.href} data={r} />}
+									element={<MatchedRoute key={r.href} data={r} api={api} />}
```

Comment:
Interesting - a context value may make sense as well (in the future)

Reply:
I had a similar thought, but it also seemed reasonable that the parent profile renderers would just get it as a prop. Could go either way.